deploy/dev-ngsa-asb-westus2/config.json

```
@@ -1,7 +1,7 @@
 {
-    "zone": "az-westus2-dev",
-    "region": "westus2",
-    "mi.name": "ngsa-asb-ngsa-id",
-    "domain" : "cse.ms",
+    "zone": "az-westus3-dev",
+    "region": "westus",
```

Should this be westus3?
```suggestion
    "region": "westus3",
```

---

yes, thank you.